openmetadata-ui/src/main/resources/ui/src/components/Glossary/GlossaryV1.component.tsx

```diff
@@ -81,19 +101,22 @@ const GlossaryV1 = ({
     [action]
   );
 
-  const fetchGlossaryTerm = async (params?: ListGlossaryTermsParams) => {
-    setIsLoading(true);
+  const fetchGlossaryTerm = async (
+    params?: ListGlossaryTermsParams,
+    refresh?: boolean
+  ) => {
+    refresh ? setIsTermsLoading(true) : setIsLoading(true);
     try {
       const { data } = await getGlossaryTerms({
         ...params,
         limit: API_RES_MAX_SIZE,
-        fields: 'tags,children',
+        fields: 'tags,children,reviewers,relatedTerms,owner,parent',
```

Comment:
It's expensive, can we fetch them instead in a modal? 
Loading all this details on Table is not necessary

Reply:
Will be addressed with hierarchy of glossary terms